### PR TITLE
msx1_cart.xml: Add 76 items

### PR DESCRIPTION
**Pull request by wilbertpol**
New working software list items
-------------------------------
Safari X (Japan, The Links) [file-hunter]
Sasa (Japan, alt 2) [file-hunter]
Scramble Eggs (Korea) [file-hunter]
Sea Horse (Arab, v1.01) [file-hunter]
Skooter (Arab) [file-hunter]
Snake It (Arab) [file-hunter]
Snake It (Japan, alt) [file-hunter]
Space Shuttle - A Journey into Space (Japan) [file-hunter] Star Trek (Japan) [file-hunter]
Star Trek (Japan, alt) [file-hunter]
Star Wars (Arab, alt) [file-hunter]
Super Bioman 1 (Korea) [file-hunter]
Super Bioman 1 (Korea, alt) [file-hunter]
Super Game 25 (Korea) [file-hunter]
Super Racer (Japan) [file-hunter]
Super Snake (Japan, alt 3) [file-hunter]
Supertripper (Spain, alt) [file-hunter]
S.o.L.o. (English) [MSXDev]
S.o.L.o. (Spanish) [MSXDev]
Sacred Valley [MSXDev]
Saimazoom (English) [MSXDev]
Saimazoom (Spanish) [MSXDev]
SD Sniper [MSXDev]
Seleniak [MSXDev]
Seleniak (alt) [MSXDev]
Sensha Game (20211107) [ochixn]
Sensha Game (20210318) [file-hunter]
Sensha Game (20201128) [file-hunter]
Shadow of the Pig (v1.3) [MSXDev]
Shadow of the Pig (v1.2) [file-hunter]
Shadow of the Pig [file-hunter]
Shadow Switcher [MSXDev]
Shmup! (v1.1) [MSXDev]
Shmup! (v1.0) [MSXDev]
Shouganai (v1.1) [MSXDev]
Shouganai (v1.0) [MSXDev]
Sink King [MSXDev]
Skate Air (English) [MSXDev]
Skate Air (Spanish) [MSXDev]
SKULLrs [MSXDev]
Slender - The Camping (v1.1) [MSXDev]
Slender - The Camping (v1.0) [MSXDev]
Slime Center [MSXDev]
Snail Maze [MSXDev]
Snake [file-hunter]
Snoopy Town [file-hunter]
Snowclimber 2K [file-hunter]
Snowclimber [file-hunter]
Soukoban Pocket Edition [MSXDev]
Sp8 Invaders [MSXDev]
Space Crussader (English) [MSXDev]
Space Crussader (Spanish) [MSXDev]
SpaceCat [MSXDev]
SPC Super HOOPS [MSXDev]
Speedjet Racers (demo) [file-hunter]
Sphere Redux [file-hunter]
Sport Racer [file-hunter]
SquareBall [MSXDev]
Stan, the Dreamer [MSXDev]
Step (v1.2) [MSXDev]
Step (v1.1) [file-hunter]
Stevedore (demo) [file-hunter]
Stratos (2005) [MSXDev]
Stratos (2004) [MSXDev]
Stray Cat (10th anniversary) [Imanok]
Stray Cat (MSX cartridge shop) [file-hunter]
Stray Cat [MSXDev]
Stupid Martians (v1.1) [MSXDev]
Stupid Martians (v1.0) [file-hunter]
Subacuatic [MSXDev]
Subacuatic (alt) [file-hunter]
SubCommander (v1.02) [MSXDev]
SubCommander (v1.01) [file-hunter]
SubCommander (v1.00) [file-hunter]
Sudoku [MSXDev]
Super Ilevan [file-hunter]

Replaced Super Bros. World 1 (Korea)
Renamed seahorse to seahorsea

### CODE REVIEW

File: hash/msx1_cart.xml [hunk 1]
```diff
@@ -11521,6 +11610,32 @@ kept for now until finding out what those bytes affect...
...
+		<year>199?</year>
+		<publisher>Zemina</publisher>
+		<info name="alt_title" value="바이오맨1"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="slot" value="ascii8" />
+			<dataarea name="rom" size="0x10000">
+				<rom name="super bio man 1 - segye-ro electronics (1992) (alt).rom" size="0x10000" crc="d9746de4" sha1="d01e5c0005c5dcdaaeafc6c11cc4ce759a4b4866" />
+			</dataarea>
+		</part>
+	</software>
```
Comment (cuavas) [inline hash/msx1_cart.xml:11637]: The title on the cover art is “슈퍼 바이오맨 1” (Super Bioman 1). The cartridge label just says “슈퍼 바이오맨” (Super Bioman).  Do the other Korean Super Bioman games also have “슈퍼” in their Korean titles?

File: hash/msx1_cart.xml [hunk 2]
```diff
@@ -17683,22 +17917,776 @@ legacy FM implementations cannot find it.
...
+		<year>2022</year>
+		<publisher>&lt;homebrew&gt;</publisher>
+		<info name="developer" value="thegeps"/>
+		<part name="cart" interface="msx_cart">
+			<feature name="start_page" value="1"/>
+			<dataarea name="rom" size="0x8000">
+				<rom name="shadow of the pig v1 - the geps (2022).rom" size="0x8000" crc="93530a29" sha1="b7b7c790e27ea6bafaa00cc1984fb9a1ee23a3cd" />
+			</dataarea>
+		</part>
+	</software>
```
Comment (cuavas) [inline hash/msx1_cart.xml:18103]: Should this one have the version in the description like the other versions do?

File: hash/msx1_cart.xml (hunk 1 above)
Comment (wilbertpol) [reply hash/msx1_cart.xml]: All the covers have "슈퍼" in their titles so it makes sense to add that to the alt_titles.
In-game only an English title is displayed. Super Bioman 1 has the '1' on the title screen.

**Comment by cuavas** [comment]
Where did the replacement “Super Bros. World 1 (Korea)” come from?

**Comment by wilbertpol** [comment]
Also from file-hunter

**Pull request merged**